### PR TITLE
[pylint] adding in correct links for pylint

### PR DESCRIPTION
**Pull request by l0lawrence**
To be merged either before first or after last checker, fixes issues with README and tests

### CODE REVIEW

**Comment by l0lawrence** [comment]
Double check accuracy of links

File: scripts/pylint_custom_plugin/README.md [hunk 1]
```diff
@@ -43,20 +43,20 @@ In the case of a false positive, use the disable command to remove the pylint er
...
-| client-incorrect-naming-convention                 | Check that you use... snake_case for variable, function, and method names. Pascal case for types. ALL CAPS for constants.                                            | # pylint:disable=client-incorrect-naming-convention                                      | [link](https://azure.github.io/azure-sdk/python_design.html#naming-conventions)         |
+| client-method-should-not-use-static-method         | Use module level functions instead.                                                                                                                                  | # pylint:disable=connection-string-should-not-be-constructor-param                       | [link](https://azure.github.io/azure-sdk/python_implementation.html#method-signatures) |
```
Comment (kristapratico) [inline scripts/pylint_custom_plugin/README.md:46]: Unrelated to your changes, but it looks like this one has the wrong name shown for the disable command. Can we update it in this PR?

**Review by kristapratico — APPROVED** [review]
Yay thanks for doing this

**Review by catalinaperalta — APPROVED** [review]
LGTM!

**Pull request merged**